index.js

```diff
@@ -287,6 +328,7 @@ function VFile(options) {
      */
 
     self.filePath = filePathFactory(self);
+    self.fileBasename = fileBasenameFactory(self);
```

Comment:
I previously mentioned `basename`. I think that’s enough of a name, so I don’t think the `file` prefix is needed. I’d say remove it, what are your thoughts?